src/ui/public/agg_types/metrics/latest.js

```diff
@@ -0,0 +1,44 @@
+import AggTypesMetricsMetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
+import orderAndSizeTemplate from 'ui/agg_types/controls/order_and_size.html';
+
+export default function AggTypeMetricLatestProvider(Private) {
+  let MetricAggType = Private(AggTypesMetricsMetricAggTypeProvider);
```

Comment:
Use `const` by default.  Only use `let` if you need to mutate the reference.  That applies to this whole file, but I won't mark each usage of `let`.